src/components/pages/ParentHome/ParentCalendar.js

```diff
@@ -67,139 +191,66 @@ function ParentCalendar(props) {
     setActiveModal(id);
   };
 
-  const showFull = date => {
-    let newDate = date._d.toLocaleString('en-US', {
-      day: '2-digit', // numeric, 2-digit
-      year: 'numeric', // numeric, 2-digit
-      month: '2-digit', // numeric, 2-digit, long, short, narrow
-    });
-    newDate = newDate.replace(/\//g, '-');
-    let newArr = schedule.sessions.filter(item => {
-      return item.start_date === newDate;
-    });
-
-    setFullProgram(newArr);
-  };
-  const handleOk = () => {
-    setActiveModal(0);
-    setIsModalVisible(false);
-  };
-
-  const handleOkFull = () => {
-    setFullProgram(false);
-  };
-
   const handleCancel = () => {
     setActiveModal(0);
-    setIsModalVisible(false);
   };
 
-  const handleCancelFull = () => {
-    setFullProgram(false);
+  const handleClick = () => {
+    return null;
   };
 
+  // value's format mm/dd/yyyy
```

Comment:
Is this comment necessary?

Reply:
yeah I think it would be beneficial for next units because the calendar value from ant design is not stated anywhere in its documents. commenting this way might help them determine the format right away.

Reply:
Yeah, it would definitely help them.  you can save time and remove any doubt by simply telling them what format to expect